### ISSUE TITLE
"ValueError: Paths don't have the same drive" for all invocations of specialist

### ISSUE THREAD

**Issue by syegulalp** [brandtbucher/specialist#8]
I get the following error no matter how I try to invoke `specialist`:

```
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "D:\Dev\conway-2023\.env\Scripts\specialist.exe\__main__.py", line 4, in <module>
    import shutil
    ^^^^^^^^^^^^^^
  File "D:\Dev\conway-2023\.env\Lib\site-packages\specialist.py", line 78, in <module>
    os.path.commonpath([sysconfig.get_path("stdlib"), sysconfig.get_path("purelib")])
  File "<frozen ntpath>", line 814, in commonpath
ValueError: Paths don't have the same drive
```

**Comment by syegulalp** [comment]
Note: I suspect this might be because I have my underlying Python installation and my venv/project on two separate drives.

**Comment by brandtbucher** [comment]
Ah, nice find. I'll try to look into this more tomorrow, but if somebody wants to dig into it more or propose a fix before then, that would be greatly appreciated!

**Comment by brandtbucher** [comment]
@syegulalp, do you mind trying out 177837c and seeing if everything works as expected? I can then cut a new release if so.

**Comment by syegulalp** [comment]
I'm now able to get `specialist` working from the command line. However, one of the projects I'm analyzing doesn't seem to be working correctly with it. I suspect that's because my project structure is a little unorthodox, so I'm going to clean that up. For now you can consider this particular issue closed, though.

**Comment by brandtbucher** [comment]
I’d like to leave this open so I’m reminded to make a release.

Please open another issue explaining the weird behavior you’re seeing… I’d like to fix it, if possible!

**Comment by syegulalp** [comment]
My project has a layout roughly like this:

```
conway.py
    life [module]
        app.py
```
where `conway.py` is executed and imports from `life`. 

`conway.py`:

```py
from life import app

app.main()
```

However, invoking this through `specialist` generates this error:

```
Traceback (most recent call last):
  File "<frozen runpy>", line 198, in _run_module_as_main
  File "<frozen runpy>", line 88, in _run_code
  File "C:\Python311\Scripts\specialist.exe\__main__.py", line 7, in <module>
  File "C:\Python311\Lib\site-packages\specialist.py", line 549, in main
    raise caught[0] from None
  File "C:\Python311\Lib\site-packages\specialist.py", line 178, in _catch_exceptions
    yield caught
  File "C:\Python311\Lib\site-packages\specialist.py", line 507, in main
    runpy.run_path(  # pylint: disable = no-member
  File "<frozen runpy>", line 291, in run_path
  File "<frozen runpy>", line 98, in _run_module_code
  File "<frozen runpy>", line 88, in _run_code
  File "conway.py", line 1, in <module>
    from life import app
ModuleNotFoundError: No module named 'life'
```

**Comment by brandtbucher** [comment]
I can reproduce the issue, yeah. For the time being, invoking `specialist` as `python -m specialist` should fix this. For example:

```sh
$ python -m specialist --targets 'life/*.py' conway.py
```

**Comment by brandtbucher** [comment]
Okay @syegulalp, the new issue should be fixed in f2831c1. Please try it out and let me know if your app works (this was a bug in `specialist`, not in your project layout). If it works for you, I'll make a new release.

Thanks for your helpful feedback!

**Comment by brandtbucher** [comment]
Thanks for the help! These fixes are shipped in [`0.5.1`](https://pypi.org/project/specialist/0.5.1).
